connector.go

```diff
@@ -13,41 +13,41 @@ type Connector interface {
 
 	// Deprecated: the method will be removed in the next major version,
 	// use a PingRequest object + Do() instead.
-	Ping() (Response, error)
+	Ping() ([]interface{}, error)
```

Comment:
It looks like several changes are not separated into commits properly. Now some of my review comments for the first commit looks outdated.